src/main/java/org/kohsuke/github/GitHubRequest.java

```diff
@@ -45,7 +45,7 @@
     private final String apiUrl;
     private final String urlPath;
     private final String method;
-    private final InputStream body;
+    private final Object body;
```

Comment:
It needs to be object, unless what you mean is that we should add a separate field for `private final Object bodyObject;` and keep the `InputStream body` as well.   I don't really see the point of the that though. You can't have both. 

The alternative would be to have to allow the addition of an object to `args` with the key `""`.  This would a signal to set the root object to passed as body.